include/nbl/video/IGPURayTracingPipeline.h

```diff
@@ -10,9 +10,9 @@
 namespace nbl::video
 {
 
-class IGPURayTracingPipeline : public IBackendObject, public asset::IRayTracingPipeline<const IGPUPipelineLayout, const IGPUShader>
+class IGPURayTracingPipeline : public IBackendObject, public asset::IRayTracingPipeline<const IGPUPipelineLayout, const asset::IShader>
 {
-        using pipeline_t = asset::IRayTracingPipeline<const IGPUPipelineLayout,const IGPUShader>;
+        using pipeline_t = asset::IRayTracingPipeline<const IGPUPipelineLayout,const asset::IShader>;
```

Comment:
the `I....Pipeline` no longer need to be templated on the shader type (there's only one now)